schema/rhcPlaybookRunResponse.message.yaml

```diff
@@ -0,0 +1,69 @@
+---
+$id: rhcPlaybookRunResponseMessage
```

Comment:
This name is not great. We have two response types:

1. Ansible runner events sent by rhc-worker-playbook plugin
2. Satellite events sent by rhc-worker-satellite plugin

As both are related to "rhc", calling this one "rhcPlaybookRunResponseMessage" is a bit confusing. I suggest to do `playbookRunResponse.message.yaml` vs `playbookSatRunResponse.message.yaml` or `playbookWorkerResponse.message.yaml` vs `satelliteWorkerResponse.message.yaml` or something similar.

Reply:
Makes sense! Used `playbookRunResponse.message.yaml` vs `playbookSatRunResponse.message.yaml` :+1: